install/ansible/README.md

```diff
@@ -47,6 +47,7 @@ The full list of configurable parameters is as follows:
 | Parameter | Description | Values |
 | --- | --- | --- |
 | `cluster_flavour` | Defines whether the target cluster is a Kubernetes or an Openshift cluster. | Valid values are `k8s` and `ocp` (default) |
+| `github_api_token` | The API token used for authentication when calling the GitHub API | Any valid GitHub API token or empty (default) |
```

Comment:
Any chance you can add Ansible documentation to the istio.io website as a separate PR and point this readme towards it?

Reply:
I have already opened this [PR](https://github.com/istio/istio.github.io/pull/896) regarding the documentation.
If this one get's accepted, I will accordingly update the documentation one as well.